runtime/src/lib.rs

```diff
@@ -143,15 +143,20 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     transaction_version: 1,
 };
 
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
+pub const MILLISECS_PER_BLOCK: u64 = 12000;
 
 pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
 
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 10 * MINUTES;
+
 // Time is measured by number of blocks.
 pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
-
```

Comment:
could we also use these in our standalone chain https://github.com/DataHighway-DHX/node/blob/master/pallets/primitives/src/constants/mod.rs#L35 ?

Reply:
@ltfschoen I am not sure what you are trying to say here. They are already there in standalone chain

Reply:
sorry yes they are, closing this